src/branch.test.ts

```diff
@@ -3,7 +3,7 @@ import {
   assertEquals,
   assertObjectMatch,
 } from 'https://deno.land/std@0.160.0/testing/asserts.ts'
-import { z } from 'https://deno.land/x/zod@v3.21.4/mod.ts'
+import { z } from 'npm:zod@3.21.4'
```

Comment:
Why the change from deno.land to npm?

Reply:
to be frank, I'm not very familiar with deno, so I might be missing how to do this better. I'd appreciate some advice

typeschema relies on optional peer dependencies to interact with the validation libs. so deno, for some reason, wasn't recognizing `https://deno.land/x/zod@v3.21.4/mod.ts` as being equivalent to `npm:zod@3.21.4` (and that was making the tests fail due to typeschema's `Schema` type not including zod as a valid schema). maybe one way to better fix this is to port typeschema to deno (instead of relying on the npm import), but I'm not sure how to make deno work with optional peer dependencies